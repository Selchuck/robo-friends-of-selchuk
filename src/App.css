h1 {
  font-family: 'Oswald', sans-serif;
  font-weight: 900;
  color: white;
  letter-spacing: 2px;
  text-decoration: underline;
  text-shadow: 2px 2px black;
}
