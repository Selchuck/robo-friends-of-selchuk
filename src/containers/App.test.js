import { shallow } from 'enzyme';
import App from './App';
import React from 'react';

let wrapper;
beforeEach(() => {
    const mockProps = {
        robots: [],
        searchField: ''
    }
    wrapper = shallow(<App  {...mockProps}/>)
})

it('it renders main page without crashing', () => {
    expect(wrapper).toMatchSnapshot();
})

it('it filters robots correctly', () => {
    expect(wrapper).toEqual({})
})


