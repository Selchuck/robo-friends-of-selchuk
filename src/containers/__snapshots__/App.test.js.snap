// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render App component 1`] = `ShallowWrapper {}`;

exports[`it renders main page without crashing 1`] = `ShallowWrapper {}`;
